test/data/Tmp.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  Tmp: JSON.stringify({
+    prisbegin: '1687363200',
+    prisend: '1688572800',
+    cavabegin: '1690761600',
+    PurchasePlatformLockEnabled: true,
+    lqo9: {
+      mt: ['Survival', 'Extermination', 'Alchemy'],
+      mv: ['UnpoweredCapsules', 'FortifiedFoes', 'AlchemicalShields'],
+      c: [
+        ['RegeneratingEnemies', 'PointBlank'],
+        ['Quicksand', 'EMPBlackHole'],
+        ['Deflectors', 'AcceleratedEnemies'],
+      ],
+      fv: ['ContactDamage', 'Withering', 'Gearless', 'Exhaustion'],
+    },
+    sfn: 554,
+  }),
+};
```

Comment:
```suggestion
export const TMP = JSON.stringify({
    prisbegin: '1687363200',
    prisend: '1688572800',
    cavabegin: '1690761600',
    PurchasePlatformLockEnabled: true,
    lqo9: {
      mt: ['Survival', 'Extermination', 'Alchemy'],
      mv: ['UnpoweredCapsules', 'FortifiedFoes', 'AlchemicalShields'],
      c: [
        ['RegeneratingEnemies', 'PointBlank'],
        ['Quicksand', 'EMPBlackHole'],
        ['Deflectors', 'AcceleratedEnemies'],
      ],
      fv: ['ContactDamage', 'Withering', 'Gearless', 'Exhaustion'],
    },
    sfn: 554,
  }),
};
```

Reply:
wouldn't work since the lint does enforces default exports on files with a single export

Reply:
ok, default export is fine too, since it's all you're using